extension/js/common/platform/xss.ts

```diff
@@ -235,6 +235,7 @@ export class Xss {
   };
 
   public static stripEmojis = (str: string) => {
+    console.log('ci test');
```

Comment:
just added and remove this temporarily to somehow fixed CI test not working correctly aftering clicking 'rerun' which doesn't produce the desired effect.